src/graphics/programlib/programlib_phong.js

```diff
@@ -277,7 +277,7 @@ pc.programlib.phong = {
             } else {
                 code += chunks.specularAaNonePS;
             }
-            code += this._addMap("specular", options, chunks, uvOffset);
+            code += this._addMap(options.metalnessWorkflow? "metalness" : "specular", options, chunks, uvOffset);

```

Comment:
Regarding internal properties like metalnessWorkflow vs useMetalness, is there a reason why they should be named differently? Isn't it more consistent to use the same names for option variables and their counterparts?